app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  DEFAULT_AVATAR_IMAGE = "default_avatar.png"
+
+  def default_avatar(user_profile)
+    user_profile.avatar.url.nil? ? DEFAULT_AVATAR_IMAGE : user_profile.avatar.url
```

Comment:
you can avoid ternary operator here and
`user_profile.avatar.url || DEFAULT_AVATAR_IMAGE` is equal